### DISCUSSION TITLE
Can we please put "sentence casing" as a requirement in the paste header component?

### DISCUSSION THREAD

**Discussion by mjanssen2021** [twilio-labs/paste#2157] [Ideas] [answered]
I saw this discussion: https://github.com/twilio-labs/paste/discussions/417 in which caroline tecks lays out all the rationale for why our Twilio content should be sentence case. 

Finding this was helpful because I could explain to the PM that we did have a standard for titles, and that this is why we wanted those standards. 

Could we put this requirement in the Twilio component documentation for header and anywhere else titles apply in the paste system? Thank you!

**Comment by shleewhite** [comment, marked as answer]
Hi @mjanssen2021, thanks for the great suggestion! I made this [PR to add the information to the docs site](https://github.com/twilio-labs/paste/pull/2159 ).

**Reply by mjanssen2021** [reply]
Thanks @shleewhite ! :)

**Reply by shleewhite** [reply]
It just merged! The changes should be live soon.

**Reply by mjanssen2021** [reply]
YAYYYY! V exciting.
